src/controllers.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable import/no-cycle */
```

Comment:
Even though you can complete the project by disabling the no-cycle linter, I highly recommend organising your code to not cause any import cycles, as these dependency cycles can lead to bugs in the future. You can do so by calling the module functions with arguments instead and using another function to feed as an argument. This is in no way the perfect solution but should help you overcome the import cycles.